src/components/Request/Pay/Views/Initial.view.tsx

```diff
@@ -12,14 +13,40 @@ import { useCreateLink } from '@/components/Create/useCreateLink'
 import { peanut, interfaces } from '@squirrel-labs/peanut-sdk'
 import TokenSelector from '@/components/Global/TokenSelector/TokenSelector'
 import { switchNetwork as switchNetworkUtil } from '@/utils/general.utils'
+import { type ITokenPriceData } from '@/interfaces'
 
 const ERR_NO_ROUTE = 'No route found to pay in this chain and token'
 
-enum RequestStatus {
+enum ViewState {
+    INITIAL = 'INITIAL',
     LOADING = 'LOADING',
-    CLAIM = 'CLAIM',
-    NOT_CONNECTED = 'NOT_CONNECTED',
-    NOT_FOUND = 'NOT_FOUND',
+    READY_TO_PAY = 'READY_TO_PAY',
+    ERROR = 'ERROR',
+}
+
+async function createXChainUnsignedTx({
```

Comment:
thought: this seems like an unnecessary function no? it literally just wraps an sdk function

Reply:
agree!